README.md

```diff
@@ -71,137 +73,123 @@ To retrieve tenant minter configuration:
+To replace one or more minter resources, first get the minter configuration and call the `elv-live tenant_replace_minter_config` command, passing in all the existing values that you don't want changed (or else they will be regenerated).
 
-For example to set the legacy_shuffle_seed and not change any of the other settings, pass in the existing values as options:
+For example to set the 'legacy\_shuffle\_seed' and not change any of the other settings, pass in the existing values as options:
 
 ```
 ./elv-live tenant_replace_minter_config itenKGHd3iedqtA39krJUPkBTCNoTeX --minter ikms37TNyoqBKqjbAPS7HMv4gzvkJzTa --mint_helper 0x3AEA63e14e084A87Cf2588Fd0987e12db71f81C1 --proxy 0x41899355fE869c370ED92eaA9f791289c4Da9F9D --proxy_owner ikms4DgAcxHqMyv6pJYqdb8wqUrmUmLk --legacy_shuffle_seed 1000
 
 ```
 
 
-### Marketplace info
+##### Marketplace info
 
-Show tenant-level marketplace information (including validation):
+To show tenant-level marketplace information (including validation):
 
 ``` bash
-./elv-live tenant_show itenYQbgk66551FEqWr95xPmHZEjmdF ilib3Drbefo66PfWvY1NVup4VZFzDJ68  iq__21pxPgnpyYkV666nZ2RhNGYGYdwC --check_cauth ikms2BxjJaireMQXHS55gAiWkuugU5gsjx --check_minter 0x59e79eFE007F5205557a646Db5cBddA82261Ca81
-```
-
-Show number of items minted by NFT contract:
-```
-./elv-live tenant_wallets iten3RmAAA7LUZdjC55agr68xPASnKxL
+./elv-live tenant_show itenYQbgk66551FEqWr95xPmHZEjmdF --check_nfts
 ```
 
-Show all wallets that have purchases in the tenancy and the corresponding email addresses:
+To show items minted by NFT contract in a given wallet for a given tenant:
 ```
 ./elv-live tenant_balance_of iten3RmQEH7LUZdjagr68xPASnKxL 0x31d979d8fcc4bfd55a081535c7aa816b67bd40c8
 ```
-The ids required for the commands are located in the Eluvio fabric browser at:
 
+To show all wallets that have purchases in the tenancy and the corresponding email addresses:
 ```
-Tenant Id:
-Properties > Drop Event > Manage > Tenant Id
-```
-```
-Library Id:
-Properties > Library Info > Library Id
-```
-```
-Tenant Object Id *
-Properties > Eluvio LIVE Tenant > Object Id
+./elv-live tenant_wallets iten3RmAAA7LUZdjC55agr68xPASnKxL
 ```
 
-## NFT commands
 
-### Set up an NFT contract 
-Set up an NFT contract and associate it with an NFT Template object:
+#### NFT commands
+
+##### Set up an NFT contract 
+To set up a tradeable NFT contract and associate it with an NFT Template object:
 
 ```
-./elv-live nft_add_contract  ilib3ErteXJcCoTapj2ZhEvMKWau6jET iq__QrxLAAJ8V1xbdPzGVMwjHTpoFKP itenYQbgk66W1BFEqWr95xPmHZEjmdF --minthelper 0x59e79eFE007F5208857a646Db5cBddA82261Ca81 --cap 100 --name "TEST NFT SERIES A" --symbol "TESTA"
+./elv-live nft_add_contract iten3h6Eo3NT3JKZss6crss9quGPdKyS iq__QrxLAAJ8V1xbdPzGVMwjHTpoFKP 100 MYTENANCY-MYNFTNAME MYTENANCY-MYNFTSYMBOL --hold 1
 ```
 
-### Set Token URI for NFT contract:
-elv-live tenant_set_token_uri <request_type> <tenant> <contract_address> <new_token_uri> [options]
+##### Build NFT metadata
+
+
```

Comment:
let's show the build command here since we show all the other ones (and mention it should be run after nft_add_contract)